validation/models.py

```diff
@@ -345,7 +346,14 @@ class Recording(models.Model):
     # TODO: determine this automatically during import process
     quality = models.CharField(
         help_text="Is the recording clean? Is it suitable to use publicly?",
-        **arguments_for_choices(QUALITY_CHOICES),
+        # **arguments_for_choices(QUALITY_CHOICES),
```

Comment:
How come this is commented out? Can this be deleted entirely?

Does `QUALITY_CHOICES` need to exist at all in this PR?

Reply:
I didn't actually write `QUALITY_CHOICES` but I can delete it